modules/ROOT/pages/editor-plugin-version.adoc

```diff
@@ -1,160 +1,121 @@
+===== Example: using the `{productmajorversion}` release channel
 
 [source,html,subs="attributes+"]
 ----
-<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/{productmajorversion}/plugins.min.js?mentions=3.0&powerpaste=6.0&advcode=sdk" referrerpolicy="origin"></script>
+<script src="{cdnurl}" referrerpolicy="origin"></script>
 ----
 
-The disadvantage of `+plugins.min.js+`: to load only one plugin from the {cloudname} and the rest from a self-hosted deployment, *ALL* other plugins need to be added as query parameter with the version as `+sdk+`. When {cloudname} releases a new plugin, this will need to be updated. In situations where most premium plugins need to be loaded from a self-hosted deployment, use `+cloud-plugins.min.js+`.
+[#{productmajorversion}-testing-release-channel]
+==== {productmajorversion}-Testing Release Channel
 
-==== cloud-plugins.min.js
+This channel deploys the current *release candidate* for the `{productmajorversion}` channel. The {productname} release candidate is undergoing quality assurance. The current version of {productname} available through the `{productmajorversion}-testing` channel can be found on the https://cdn.tiny.cloud/1/no-api-key/tinymce/{productmajorversion}-testing/version.txt[{cloudname} {productname} {productmajorversion}-testing version page].
 
-Instead of loading `+tinymce.min.js+` from {cloudname}, serve {productname} from a self-hosted server, and load `+cloud-plugins.min.js+` from {cloudname}. Unlike `+plugins.min.js+`, `+cloud-plugins.min.js+` defaults to loading every *premium* plugin from the *self-hosted {productname} installation*, not {cloudname}. However, plugins can be loaded from {cloudname} by specifying them as query parameters.
+[#example-using-the-{productmajorversion}-testing-release-channel]
+===== Example: Using the `{productmajorversion}-testing` Release Channel
 
-With `+cloud-plugins.min.js+`, the plugins listed in the query strings do not require a version. If there is no version specified, {productname} uses the version bundled with the {productname} version requested. There is also no need to specify `+sdk+` as the version for any plugin, as that is the default.
+[source,html,subs="attributes+"]
+----
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/{productmajorversion}-testing/tinymce.min.js" referrerpolicy="origin"></script>
+----
+
+[#{productmajorversion}-dev-release-channel]
+==== {productmajorversion}-Dev Release Channel
 
-The following example:
+This channel deploys nightly builds of {productname}, which includes unreleased changes documented in the https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/CHANGELOG.md[{productname} changelog]. The current version of {productname} available through the `{productmajorversion}-dev` channel can be found on the https://cdn.tiny.cloud/1/no-api-key/tinymce/{productmajorversion}-dev/version.txt[{cloudname} {productname} {productmajorversion}-dev version page].
```

Comment:
The current dev versions is 7.2, but the changelog link mentioned here doesn't include 7.2 changes. Is this statement still correct?

Reply:
@ltrouton Can you advise if the above is still accurate based on changes leading up to `7.0`

Reply:
the in-repo changelog is no longer updated before the next release. I don't know if we publish the "unreleased changelog" anywhere now.

Maybe just change `documented in the changelog` to `from the tinymce repository`. Link to the repo (noting we use branch `main` now not `develop`).

Reply:
```suggestion
This channel deploys nightly builds of {productname}, which includes **unreleased changes** from the link:https://github.com/tinymce/tinymce/blob/main/modules/tinymce/CHANGELOG.md[{productname} changelog^]. 

The current version of {productname} is available on the `{productmajorversion}-dev` channel can be found on the link:https://cdn.tiny.cloud/1/no-api-key/tinymce/{productmajorversion}-dev/version.txt[{cloudname} {productname} {productmajorversion}-dev version page^].

[NOTE]
{productname} now uses `main` branch for nightly builds instead of `develop` as of {productname} `7.0.0`.
```

Reply:
That still links to the changelog. How about

```
This channel deploys nightly builds of {productname}, which includes **unreleased changes** from the link:https://github.com/tinymce/tinymce/tree/main[{productname} repository^]. 
```